.github/workflows/create-github-release.yml

```diff
@@ -0,0 +1,37 @@
+name: create-github-release
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      # There's no event type for "merged", so we just run any time a PR is closed, and exit early
+      # if the PR wasn't actually merged.
+      - closed
+
+jobs:
+  create-github-release:
+    # Since the workflow runs any time a PR against main is closed, we need this
+    # `if` to make sure that the workflow only does anything meaningful if the PR
+    # was actually merged.
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v4
+        with:
+          ref: main
+      - name: Get version property
+        id: get-version-property
+        run: |
+          PACKAGE_VERSION=$(jq -r ".version" package.json)
+          echo "package_version=$PACKAGE_VERSION" >> "$GITHUB_OUTPUT"
+      - name: Create github release
+        uses: softprops/action-gh-release@v2
+        with:
+          tag_name: v${{ steps.get-version-property.outputs.package_version }}
+          name: v${{ steps.get-version-property.outputs.package_version }}
+          body: See [release notes](https://developer.salesforce.com/docs/platform/salesforce-code-analyzer/guide/release-notes.html)
+          token: ${{ secrets.SVC_CLI_BOT_GITHUB_TOKEN }}j
```

Comment:
This is an org-level secret, so we'll need to ask the CLI team to grant it to us, which they can do.

Reply:
OK.

This code looks good - but I'd wait to merge until this secret has been granted to our org so that we can qa this.

Reply:
Agreed.

Reply:
@stephen-carter-at-sf , the secret has been shared with us, so we can merge this as soon as I either implement or abandon the use of `gh pr merge`.